android/src/main/java/io/wifi/p2p/WiFiP2PManagerModule.java

```diff
@@ -90,6 +90,7 @@ public void onGroupInfoAvailable(WifiP2pGroup group) {
   @ReactMethod
   public void init(Promise promise) {
     if (manager != null) { // prevent reinitialization
+      promise.resolve(manager != null && channel != null);
```

Comment:
Maybe better to reject a promise with a message, that you are trying to reinitialize already initialized module? What do you think?

Reply:
I may turn the wifi on and off multiple times, but I'm not sure if I need to `initialize` again After the wifi is turned back on, when I call `initialize` again, it gets stuck

Reply:
@tyanbiao okay, let's leave it as it is right now then :)